src/index.js

```diff
@@ -51,7 +34,7 @@ export const isPluginRequired = (supportedEnvironments, plugin) => {
       const lowestTargetedVersion = supportedEnvironments[environment];
 
       if (typeof lowestTargetedVersion === "string") {
-        throw new Error(`Target version must be a number, 
+        throw new Error(`Target version must be a number,
           '${lowestTargetedVersion}' was given for '${environment}'`);
```

Comment:
Yes! I tried not to touch index.js anywhere that wasn't related to extracting option parsing but if this PR goes through I'd love to follow up with one to improve all logging and error messages like this.